base/applications/mspaint/winproc.cpp

```diff
@@ -575,6 +576,100 @@ LRESULT CMainWindow::OnSysColorChange(UINT nMsg, WPARAM wParam, LPARAM lParam, B
     return 0;
 }
 
+typedef ULONG (WINAPI *FN_MAPISendMail)(LHANDLE, ULONG_PTR, lpMapiMessage, FLAGS, ULONG);
+typedef ULONG (WINAPI *FN_MAPISendMailW)(LHANDLE, ULONG_PTR, lpMapiMessageW, FLAGS, ULONG);
+
+ULONG CMainWindow::OpenMailerWithAttachmentW(LPCWSTR pszPathName)
+{
+    CStringW strFileTitle;
+    if (PathFileExistsW(pszPathName)) // File does exist?
+    {
+        strFileTitle = PathFindFileNameW(pszPathName);
+    }
+    else // Not existing
```

Comment:
Does this cover the case of the file that already exists, *but* has been modified in the meantime (and not saved) before sending it by mail?

Reply:
It may need confirmation if save or not.